test/.weeder.yaml

```diff
@@ -77,3 +77,24 @@
       - module:
         - name: Dir.Unused
         - identifier: unused
+
+## TODO: These false positives are due to the use of Cabal internal
+## libraries, which are not yet supported by Weeder.
```

Comment:
Is this comment true after this patch is merged?

Reply:
Good question, I have no idea!